docs/development.md

```diff
@@ -39,16 +39,22 @@ git clone https://github.com/<your-github-username>/autorest.go.git
 
 Once you have the code locally, you can build it.
 
-First, update all dependencies.
+First, install all dependencies.
 
 ```terminal
-rush update
+pnpm install
 ```
 
 Then, build the code.
 
 ```terminal
-rush build
+pnpm build
```

Comment:
looks like we could only run `pnpm build:all` outside the inner package.

Reply:
fixed in [44a86ad](https://github.com/Azure/autorest.go/pull/1552/commits/44a86adea428f9e8c59053a744cea5bbc303ba6e)